seattle_example/seattle_tasks.yml

```diff
@@ -69,18 +72,24 @@
 #
 - name: Ingest census blocks to postgres
   command: ogr2ogr -f PostgreSQL -t_srs EPSG:4326 -nln seattle_blkgrpce10 -nlt MultiPolygon 'PG:user=postgres dbname=piecewise' /opt/piecewise_web/seattle_blkgrpce10/CENSUS10_blkgrp_WGS.shp
```

Comment:
I wondered about making that line more readable and just discovered the folded scalars in one of the bocoup playbooks. I'll update it.